.github/workflows/demo-testing.yml

```
@@ -387,15 +421,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Test Hub with Koncur
-        uses: konveyor/ci/koncur-tackle-hub@main
+        uses: konveyor/ci/koncur-tackle-hub@feature/262-refactor-generic-provider
         with:
           image_pattern: |
             *{tackle-keyloak-init,tackle2-*,provider}--${{ github.run_id }}
           ref: ${{ github.base_ref || inputs.ref || github.ref_name}}
```

actionlint (1.7.12)

[error] property "ref" is not defined in object type {}

(expression)